packages/file_selector/file_selector_ios/ios/file_selector_ios/Sources/file_selector_ios/FileSelectorPlugin.swift

```diff
@@ -64,14 +69,12 @@ public class FileSelectorPlugin: NSObject, FlutterPlugin, FileSelectorApi {
     documentPicker.allowsMultipleSelection = config.allowMultiSelection
     documentPicker.delegate = completionBridge
 
-    let presenter =
-      self.viewPresenterOverride ?? UIApplication.shared.delegate?.window??.rootViewController
```

Comment:
We could keep this override structure instead of adding `ViewPresenterProvider`, but I thought it was cleaner to reduce test branching in production code.